README.md

```diff
@@ -1,212 +1,217 @@
+## Prerequisites
 
-4. Verify the signed image
+* RHEL x86\_64 9.2 or greater.
+* Command-line access to the Ansible control node with a user that has `sudo` privileges.
+* Installation and configuration of Ansible on a control node to perform the automation.
+* Installation of the Ansible collections on the control node, unless installing from the Ansible Automation Hub.
+  > [!NOTE]
+  To install the required dependencies automatically, you can run the `ansible-galaxy install` command.
+* An OpenID Connect (OIDC) provider, such as [Keycloak](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/sso/).
+* The ability to resolve the ingress host names, by using the Domain Name System (DNS) or the `/etc/hosts` file.
+* Optional.
+  Installation of the `podman` and [`cosign`](https://github.com/sigstore/cosign) binaries to verify that the RHTAS service is working as expected.
+
+## Deploying
+
+1. Create an `inventory` file with a single node under the `rhtas` group:
+   
+   ```
+   [rhtas]
+   123.123.123.123
+   ```
+
+2. Create a simple Ansible Playbook named `play.yml`, and replace `TODO` with your relevant information:
+   
+   ```yaml
+   - hosts: rhtas
+     vars:
+       base_hostname: TODO # e.g. example.com
+       # access credentials for registry.redhat.io (https://access.redhat.com/RegistryAuthentication)
+       tas_single_node_registry_username: TODO
+       tas_single_node_registry_password: TODO
+       tas_single_node_oidc_issuers:
+         - issuer: TODO # your OIDC provider (e.g. keycloak) URL
+           client_id: trusted-artifact-signer
+           url: TODO # your OIDC provider (e.g. keycloak) URL
+           type: email
+     tasks:
+       - name: Include TAS single node role
+         ansible.builtin.include_role:
+           name: redhat.artifact_signer.tas_single_node
+         vars:
+           ansible_become: true
+   ```
+
+3. Install the RHTAS Ansible collection.
+   
+   - If installing from Ansible Automation Hub, then run the following command:
+   
+     ```shell
+     $ ansible-playbook -i inventory play.yml
+     ```
+
+   - If running from a locally-cloned Git repository, then run the following command:
+   
+     ```shell
+     $ export ANSIBLE_ROLES_PATH="roles/" ; ansible-playbook -i inventory play.yml
+     ```
+
+4. Add the root certificate authority (CA) to your local truststore:
+  
+   ```shell
+   $ sudo openssl x509 -in ~/Downloads/root-cert-from-browser -out tas-ca.pem --outform PEM
+   $ sudo mv tas-ca.pem /etc/pki/ca-trust/source/anchors/
+   $ sudo update-ca-trust
+   ```
+   > [!TIP]
+   The certificate can be downloaded from the Certificate Viewer by navigating to `https://rekor.<base_domain>` in a web browser.
+   Download the _root_ certificate that issued the Rekor certificate.
+
+## Verifying the deployment by signing a test container
+
+1. Export the following environment variables, replacing `TODO` with your relevant information:
```

Comment:
We should also note here that the `/etc/hosts` entries need to be created - same as above.

Reply:
See my comment above.